File: Application/Common/Models/QueryParams/TicketQueryParams.cs
namespace Application.Common.Models.QueryParams
{
    public class TicketQueryParams : QueryParams
    {
        public int PassengerId { get; set; }
        public int TrainId { get; set; }
        public int DistanceFareId { get; set; }
        public int CarriageId { get; set; }
        public int SeatId { get; set; }
        public int ScheduleId { get; set; }
        public int PaymentId { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}

File: WebApi/AdminApp/src/app/@models/params/ticketQueryParams.ts
import {QueryParams} from './queryParams';

export interface TicketQueryParams extends QueryParams {
     passengerId: number;
     trainId: number;
     distanceFareId: number;
     carriageId: number;
     seatId: number;
     scheduleId: number;
     paymentId: number;
     createdAt: string;
}
